tests/_test_mscolab/test_server.py

```diff
@@ -504,8 +504,8 @@ def test_import_permissions(self):
             # creator is not listed
             assert data["success"] is True
 
-    def _create_operation(self, test_client, userdata=None, path="firstflight", description="simple test", active=True,
-                          content=None):
+    def _create_operation(self, test_client, userdata=None, path="firstflight", description="simple test",
+                          active=True, content):
```

Comment:
when you don't have a default value for content, then you break the order of arguments

SyntaxError: non-default argument follows default argument

That is the reason for many errors in the tests

Reply:
My IDE PyCharm shows that problem, like

![Bildschirmfoto 2025-04-05 um 09 53 16](https://github.com/user-attachments/assets/a6c17538-1006-4b0b-909a-8b59abb7a536)

Reply:
when you remove None here, you need to refactor all the tests where content now is unfilled too

Reply:
I won't do that. 

But when it is None, that just means you need to send the data which was in the past in the stub_code. Because now you intitalize the server with data.